frontend/src/features/admin-form/settings/components/FormEmailSection.tsx

```diff
@@ -5,43 +5,85 @@ import {
   useForm,
   useFormContext,
 } from 'react-hook-form'
-import { BiBulb } from 'react-icons/bi'
-import { Flex, FormControl, Icon } from '@chakra-ui/react'
+import { FormControl } from '@chakra-ui/react'
 import { get, isEmpty, isEqual } from 'lodash'
 import isEmail from 'validator/lib/isEmail'
 
-import { PaymentChannel } from '~shared/types'
 import {
   EmailFormSettings,
   FormResponseMode,
-  FormStatus,
   StorageFormSettings,
 } from '~shared/types/form'
 
-import {
-  GUIDE_FORM_MRF,
-  GUIDE_PREVENT_EMAIL_BOUNCE,
-  OGP_PLUMBER,
-} from '~constants/links'
-import { useMdComponents } from '~hooks/useMdComponents'
+import { GUIDE_PREVENT_EMAIL_BOUNCE } from '~constants/links'
 import {
   OPTIONAL_ADMIN_EMAIL_VALIDATION_RULES,
   REQUIRED_ADMIN_EMAIL_VALIDATION_RULES,
 } from '~utils/formValidation'
 import FormErrorMessage from '~components/FormControl/FormErrorMessage'
 import FormLabel from '~components/FormControl/FormLabel'
-import InlineMessage from '~components/InlineMessage'
-import { MarkdownText } from '~components/MarkdownText'
 import { TagInput } from '~components/TagInput'
 
 import { useMutateFormSettings } from '../mutations'
 import { useAdminFormSettings } from '../queries'
 
 interface EmailFormSectionProps {
+  isDisabled: boolean
   settings: EmailFormSettings | StorageFormSettings
 }
 
-export const EmailFormSection = ({
+interface AdminEmailRecipientsInputProps {
+  onSubmit: (params: { emails: string[] }) => void
+}
+
+const AdminEmailRecipientsInput = ({
+  onSubmit,
+}: AdminEmailRecipientsInputProps): JSX.Element => {
+  const { getValues, setValue, control, handleSubmit } = useFormContext<{
+    emails: string[]
+    isRequired: boolean
+  }>()
+
+  const { data: settings } = useAdminFormSettings()
+
+  const tagValidation = useMemo(() => isEmail, [])
```

Comment:
This doesn't seem to be doing anything special other than to reassign to a different variable name? 🤔